docs/concepts/workloads/controllers/daemonset.md

```diff
@@ -88,7 +88,7 @@ controller will think that those Pods were created by it.  Kubernetes will not s
 this.  One case where you might want to do this is manually create a Pod with a different value on
 a node for testing.
 
-If you attempt to create a DaemonSet such that 
+If you attempt to create a DaemonSet such that ???
```

Comment:
my guess is that the original text was : 

> If you attempt to create a DaemonSet that runs pods on only some nodes

then the line was broken into the way it is today. So... I'd suggest we remove this line.